test/e2e/rep.e2e.js

```diff
@@ -126,15 +128,168 @@ view "18KeRwuF6IsJGtmPAkO9IPZrOGD9xy0I/ku/APQ=="`))
     before(function() {
       return retrieve.materials.then((materials) => {
         _.each(materials, (material) => {
-          const command = `clear\nrep 1 m=QS mt=${material.id}`;
           suite.addTest(it(`set ${material.name} material`, function() {
-            return page.runScript(command)
+            return page.runScript(`clear\nrep 1 m=QS mt=${material.id}`)
               .then(() => page.waitUntilRepresentationIs(1, 'QS', 'EL'))
               .then(() => golden.shouldMatch(`1aid_QS_EL_${material.id}`, this));
           }));
         });
       });
     });
+
+  });
+
+  describe('check correct color parameters work', function() {
+
+    it('load 4V4F with an appropriate orientation and scale', function() {
+      return page.runScript(`\
+clear
+load "mmtf:4v4f"
+preset small
+view "1HM0iwbqdTz5fQpPCZaEBPfgFibut+Q0/vYEbwA=="`)
+        .then(() => page.waitUntilTitleContains('4V4F'))
+        .then(() => page.waitUntilRebuildIsDone())
+        .then(() => golden.shouldMatch('4v4f', this));
+    });
+
+    it('prepare the molecule', function() {
+      return page.runScript(`\
+set autobuild false
+rep 0 s = "chain AA" m = "CA" c = "HY" mt = "SF"
+rep 1 s = "chain AB" m = "BS" c = "OC" mt = "SF"
+rep 2 s = "chain AD" m = "LC" c = "TM" mt = "SF"
+rep 3 s = "chain AC" m = "TU" c = "CO" mt = "SF"
+rep 4 s = "chain AF" m = "CA" c = "MO" mt = "SF"
+rep 5 s = "chain AH" m = "CA" c = "UN" mt = "SF"
+rep 6 s = "chain AI" m = "CA" c = "SQ" mt = "SF"
+rep 7 s = "chain AJ" m = "BS" c = "EL" mt = "SF"`)
+        .then(() => page.runScript('build all'))
+        .then(() => page.waitUntilRebuildIsDone())
+        .then(() => golden.shouldMatch('4v4f_precolors', this));
+    });
+
+    it('change color parameters', function() {
+      return page.runScript(`\
```

Comment:
Please consider adding `set autobuild false` to the beginning of each script which depends on `build all` command. Just for the sake of clarity.